src/resources/Availability.js

```diff
@@ -0,0 +1,21 @@
+import Resource from './Resource'
+
+export default class Availability extends Resource {
+  source = 'availability';
+
+  async list ({
```

Comment:
Is this retrieving the availability of a single venue? Then it should be `retrieve(params`